go.mod

```diff
@@ -3,6 +3,7 @@ module github.com/DataDog/test-infra-definitions
 go 1.20
 
 require (
+	github.com/DataDog/datadog-agent/pkg/util/pointer v0.49.0
```

Comment:
Should not be a deps.

Reply:
Fixed: https://github.com/DataDog/test-infra-definitions/pull/463/commits/e10a97f9b0aa648e0796e5957e3d73b61c584c9a